Description: Mirgation from package src:kcm-gtk
 This patch provides smooth migration from package src:kcm-gtk to new
 package src:kde-gtk-config. Old configuration file ~/.gtkrc-2.0-kde
 is moved to ~/.gtkrc-2.0.
Author: Boris Pek <tehnick-8@mail.ru>
Last-Update: 2012-06-04

--- a/src/gtkconfigkcmodule.cpp
+++ b/src/gtkconfigkcmodule.cpp
@@ -72,6 +72,17 @@
     ui->cb_icon->setModel(m_iconsModel);
     ui->cb_icon_fallback->setModel(m_iconsModel);
     
+    QFile oldConfigFile(QDir::homePath()+"/.gtkrc-2.0-kde");
+    QFile newConfigFile(QDir::homePath()+"/.gtkrc-2.0");
+    if(oldConfigFile.exists()) {
+        newConfigFile.remove();
+        if( !oldConfigFile.rename(newConfigFile.fileName()) )
+            kDebug() << "Could not rename old config: " + oldConfigFile.fileName();
+        else
+            kDebug() << "Old config " + oldConfigFile.fileName() +
+                        "was successfully renamed to " + newConfigFile.fileName();
+    }
+    
     m_tempGtk2Preview = KGlobal::dirs()->saveLocation("tmp", "gtkrc-2.0", false);
     m_tempGtk3Preview = KGlobal::dirs()->saveLocation("tmp", ".config/gtk-3.0/settings.ini", false);
     
